pkg/openscap/openscap.go

```diff
@@ -142,7 +142,16 @@ func (s *defaultOSCAPScanner) getInputCVE(dist int) (string, error) {
 	}
 	defer out.Close()
 
-	resp, err := http.Get(cveURL.String())
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", cveURL.String(), nil)
+	if err != nil {
+		return "", fmt.Errorf("Could not request file %s:\n", err)
+	}
+
+	// Some servers require 'Accept' and 'User-Agent' headers.
+	req.Header.Add("Accept", `application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`)
+	req.Header.Add("User-Agent", `ImageInspector/latest`)
```

Comment:
Well, if we are making a separate backport commit, consider putting here ImageInspector/2.4.
Not that we'll maintain this consistently in all versions...